package.json

```diff
@@ -68,6 +68,7 @@
     "lodash.debounce": "^4.0.8",
     "lodash.noop": "^3.0.1",
     "path-to-regexp": "^1.7.0",
+    "query-string": "^6.13.1",
```

Comment:
No need for this. `url-parse` already exposes a version and we should use that instead of adding another dependency

Reply:
Thanks for the heads up. I've removed `query-string` dep.